src/NonceManager.sol

```diff
@@ -22,6 +22,7 @@ abstract contract NonceManager is INonceManager, BaseAuthorization {
         }
         nonceSequenceNumber[key] = targetSeq;
         emit NonceInvalidated(newNonce);
+        emit NonceSequenceNumberUpdated(key, targetSeq);
```

Comment:
seems duplicative with the above

Reply:
same here - the nonce gets updated on line 44, so wasn't sure if it's helpful to make them the same event or not

Reply:
What about removing `NonceInvalidated`? This func is updating the nonce to a new key + seq

Reply:
Could also do the same here, keeping NonceInvalidated and assuming that indexers will be able to support both

Reply:
yeah could do the same here - only thing is that `NonceInvalidated` only emits the newNonce, not the key and seq which might be annoying